src/main/scala/chiseltest/experimental/ChiselTestShell.scala

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Regents of the University of California (Regents)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package chiseltest.experimental
+
+import chisel3.stage.ChiselCli
+import chiseltest.internal.{TreadleBackendAnnotation, VerilatorBackendAnnotation, WriteVcdAnnotation}
+import firrtl.options.Shell
+import firrtl.stage.FirrtlCli
+
+trait ChiselTestCli extends ChiselCli with FirrtlCli {
```

Comment:
What is this and how is this used? Should there be example code in the scaladoc?

Reply:
This only used internally right now to parse the command line options strings in to a Seq of annotations. It attaches the ChiselTest specific options to options available from the upstream stack (chisel, treadle, firrtl)

Reply:
ok, I see it now